index.js

```diff
@@ -19,6 +19,7 @@ const createUnboxTransform = (queryParam) => {
 
 function ServeBlobs(sbot, config) {
   const corsEnabled = get(config, 'serveBlobs.cors', false)
+  const blobCSP = get(config, 'serveBlobs.csp', 'default-src none; sandbox')
```

Comment:
It might be good to extract all default config into one place like `default-config.js`

then pull into that : port, cors, csp